server/middlewares/stats.js

```diff
@@ -41,22 +41,52 @@ const parseTxBodyOutAmount = (txBody) => {
   }
 }
 
-const incrCountersBy = (key, value) => {
-  const [year, month, day] = getSlicedDate()
-  client.incrby(`${key}:total`, value)
-  client.incrby(`${key}:monthly:${year}-${month}`, value)
-  client.incrby(`${key}:daily:${year}-${month}-${day}`, value)
+const initVisitor = (trackingID) => {
+  return ua({
+    tid: trackingID,
+    cid: 555, //google analytics anonymous visitor
+    strictCidFormat: false, //to accept cid integer instead of UUID v4 format
+  })
+}
+
+const trackPageView = (argsObj) => {
+  const visitor = initVisitor(backendConfig.ADALITE_GA_TRACKING_ID)
+  const args = {
+    dp: argsObj.path,
+    dt: argsObj.hostname,
+    dh: argsObj.title,
+  }
+
+  visitor.pageview(args).send()
+}
+
+const trackEvent = (argsObj) => {
+  const visitor = initVisitor(backendConfig.ADALITE_GA_TRACKING_ID)
+  const args = {
```

Comment:
I would not declare a separate `args` variable but I would directly pass the object to `visitor.event`